docs/plotting.md

```diff
@@ -2,100 +2,182 @@
+
+* Main plot with candlestics and indicators following price (sma/ema)
+* Volume bars
+* Additional indicators as specified by `--indicators2`
+
+![plot-dataframe](assets/plot-dataframe.png)
+
+Possible arguments:
+
+```
+usage: freqtrade plot-dataframe [-h] [-p PAIRS [PAIRS ...]]
+                                [--indicators1 INDICATORS1 [INDICATORS1 ...]]
+                                [--indicators2 INDICATORS2 [INDICATORS2 ...]]
+                                [--plot-limit INT] [--db-url PATH]
+                                [--trade-source {DB,file}] [--export EXPORT]
+                                [--export-filename PATH]
+                                [--timerange TIMERANGE]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -p PAIRS [PAIRS ...], --pairs PAIRS [PAIRS ...]
+                        Show profits for only these pairs. Pairs are space-
+                        separated.
+  --indicators1 INDICATORS1 [INDICATORS1 ...]
+                        Set indicators from your strategy you want in the
+                        first row of the graph. Space-separated list. Example:
+                        `ema3 ema5`. Default: `['sma', 'ema3', 'ema5']`.
+  --indicators2 INDICATORS2 [INDICATORS2 ...]
+                        Set indicators from your strategy you want in the
+                        third row of the graph. Space-separated list. Example:
+                        `fastd fastk`. Default: `['macd', 'macdsignal']`.
+  --plot-limit INT      Specify tick limit for plotting. Notice: too high
+                        values cause huge files. Default: 750.
+  --db-url PATH         Override trades database URL, this is useful in custom
+                        deployments (default: `sqlite:///tradesv3.sqlite` for
+                        Live Run mode, `sqlite://` for Dry Run).
+  --trade-source {DB,file}
+                        Specify the source for trades (Can be DB or file
+                        (backtest file)) Default: file
+  --export EXPORT       Export backtest results, argument are: trades.
+                        Example: `--export=trades`
+  --export-filename PATH
+                        Save backtest results to the file with this filename
+                        (default: `user_data/backtest_results/backtest-
+                        result.json`). Requires `--export` to be set as well.
+                        Example: `--export-filename=user_data/backtest_results
+                        /backtest_today.json`
+  --timerange TIMERANGE
+                        Specify what timerange of data to use.
 
-``` bash
-python3 script/plot_dataframe.py [-h] [-p pairs]
 ```
 
-Example
+Example:
 
 ``` bash
-python3 scripts/plot_dataframe.py -p BTC/ETH
+freqtrade plot-dataframe -p BTC/ETH
 ```
 
-The `-p` pairs argument can be used to specify pairs you would like to plot.
+The `--pairs` argument can be used to specify pairs you would like to plot.
```

Comment:
yes - but i guess apart from this file that's beyond the scope of this PR ...

Reply:
On the other hand - the following looks odd (and cannot be copy/pasted):

```
This can be done by adding `-s/--strategy ClassName` to the command.
```

Reply:
> On the other hand - the following looks odd (and cannot be copy/pasted):
> 
> ```
> This can be done by adding `-s/--strategy ClassName` to the command.
> ```

`This can be done by adding `-s ClassName`/`--strategy ClassName` to the command.`